config/theme-tabler.php

```diff
@@ -70,20 +59,31 @@
         'showColorModeSwitcher' => true,
 
         /**
-         * When true, colors will display a beautiful Backpack skin :)
+         * When set to true colors are overridden
+         * @uses resources/assets/css/colors.css
          */
-        'useBackpackColorSkin' => true,
+        'overrideColorSkin' => true,
```

Comment:
```suggestion
        'extraCss' => [
            base_path('vendor/backpack/theme-tabler/resources/assets/css/colors.css'),
            base_path('vendor/backpack/theme-tabler/resources/assets/css/skin.css'),
        ],
```

Reply:
Let's use the existing `styles`  config for this, so that the devs can use their own colors if they want. See `config/ui.php` for details.